test/server/models/Update.test.js

```diff
@@ -195,16 +198,60 @@ describe('server/models/Update', () => {
         const usersToNotify = await update.getUsersToNotify();
         const receivedEmails = usersToNotify.map(u => u.email);
 
-        const expectAllEmailsFrom = usersList => {
-          return usersList.forEach(user => expect(receivedEmails).to.include(user.email));
-        };
-
-        expectAllEmailsFrom(parentCollectiveAdmins);
-        expectAllEmailsFrom(collectiveAdmins);
-        expectAllEmailsFrom(individualBackersUsers);
-        expectAllEmailsFrom(getOrganizationAdminUsers());
+        expectAllEmailsFrom(parentCollectiveAdmins, receivedEmails);
+        expectAllEmailsFrom(collectiveAdmins, receivedEmails, receivedEmails);
+        expectAllEmailsFrom(individualBackersUsers, receivedEmails);
+        expectAllEmailsFrom(getOrganizationAdminUsers(), receivedEmails);
         expect(usersToNotify.length).to.eq(expectedPrivateTotal);
       });
+
+      it('Notifies child collective users when parent collective public update is made', async () => {
+        collective.remove;
+        const update = await fakeUpdate({ CollectiveId: parentCollective.id, isPrivate: false });
+        const usersToNotify = await update.getUsersToNotify();
+        const receivedEmails = usersToNotify.map(u => u.email);
+
+        expectAllEmailsFrom(parentCollectiveAdmins, receivedEmails);
+        expectAllEmailsFrom(parentCollectiveBackers, receivedEmails);
+        expectAllEmailsFrom(parentCollectiveFollowers, receivedEmails);
+        expectAllEmailsFrom(individualBackersUsers, receivedEmails);
+        expectAllEmailsFrom(getOrganizationAdminUsers(), receivedEmails);
+
+        // Number of collective admins who are backers
+        const numberOfCollectiveAdminsAddedToBackers = 2;
+
+        expect(usersToNotify.length).to.eq(
+          parentCollectiveAdmins.length +
+            parentCollectiveBackers.length +
+            parentCollectiveFollowers.length +
+            collectiveFollowers.length +
+            individualBackersUsers.length +
+            countAdminsOfMemberOrganizations() +
+            numberOfCollectiveAdminsAddedToBackers,
```

Comment:
Shouldn't that be:

```suggestion
            countAdminsOfMemberOrganizations() -
            numberOfCollectiveAdminsAddedToBackers,
```

The goal of this "admins being backers" thing is to make sure we don't double count them when they have multiple roles. If I have 2 admins that are also backers, I expect the list to have 2 entries, not 4. So `admins.length + backers.length - numberOfCollectiveAdminsAddedToBackers`.